debian/stretch/foreman/dynflowd.init

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash
+#! /bin/sh
```

Comment:
bash is the one you want here, right?

Reply:
I think so I took the one from tasks & made nearly no changes https://github.com/theforeman/foreman-packaging/blob/deb/develop/plugins/ruby-foreman-tasks/debian/init.d

Reply:
k, just found both lines confusing :)